internal/gcsx/storage_handle_test.go

```diff
@@ -0,0 +1,65 @@
+package gcsx
+
+import (
+	"context"
+	"testing"
+
+	"golang.org/x/oauth2"
+)
+
+func TestNewStorageHandleHttp2Disabled(t *testing.T) {
+	sc := storageClientConfig{disableHTTP2: true,
+		maxConnsPerHost:     10,
+		maxIdleConnsPerHost: 100,
+		tokenSrc:            oauth2.StaticTokenSource(&oauth2.Token{})}
+
+	handleCreated, err := GetStorageClientHandle(context.Background(), sc)
+
+	if err != nil {
+		t.Errorf("Handle creation failure")
+	}
+	if nil == handleCreated {
+		t.Fatalf("Storage handle is null")
+	}
+	if nil == handleCreated.client {
+		t.Fatalf("Storage client handle is null")
+	}
```

Comment:
you can refactor these 3 assertions into a new method and call from all test cases.